www/src/app/controls/admin/admin-forms/admin-administrators-form/admin-administrators-form.component.ts

```diff
@@ -214,15 +235,16 @@ export class AdminAdministratorsFormComponent implements OnInit {
       }
     } else {
       this.administratorForm.markAsTouched();
+      for (let key in this.administratorForm.controls) {
+        this.administratorForm.controls[key].markAsTouched();
```

Comment:
It's so the red highlighting will show when you submit/press enter inside the form without touching the fields. 

Basically due to weirdness with the nested form, even though the form is submitted, it doesn't changed `submitted` to true, so the highlights will never show up if they aren't touched. 

There's some weirdness overall with the nested form that I'd like to revisit before we start making all of the forms reusable. 

Another is that we still get a changed after checked when you click the 'Change Password' button, because we are changing the validators. I ended up timeboxing myself and deciding that it's fine for now. It's not a bug and everything works correctly.